com.microsoft.mrtk.input/Simulation/Devices/SimulatedController.cs

```diff
@@ -498,84 +490,87 @@ private void ApplyState(ControllerControls controls)
+                    Quaternion localLookAt = Quaternion.LookRotation(Camera.main.transform.localPosition - localPose.position);
                     simulatedControllerState.deviceRotation = Smoothing.SmoothTo(
                         simulatedControllerState.deviceRotation,
-                        rigLocalLookAtCamera,
+                        localLookAt,
                         0.1f,
                         MoveSmoothingTime);
                 }
                 else if (rotationMode == ControllerRotationMode.CameraAligned)
                 {
-                    Quaternion worldCameraForward = Quaternion.LookRotation(Camera.main.transform.forward);
-                    Quaternion rigLocalCameraForward = Quaternion.Inverse(PlayspaceUtilities.ReferenceTransform.rotation) * worldCameraForward;
                     simulatedControllerState.deviceRotation = Smoothing.SmoothTo(
                         simulatedControllerState.deviceRotation,
-                        rigLocalCameraForward,
+                        Camera.main.transform.localRotation,
                         0.1f,
                         MoveSmoothingTime);
                 }
                 else
                 {
-                    simulatedControllerState.deviceRotation = rotation;
+                    simulatedControllerState.deviceRotation = localPose.rotation;
                 }
 
                 // Then set the pointer pose.
                 if (shouldUseRayVector && Handedness.ToXRNode().HasValue && HandSubsystem != null &&
-                    HandSubsystem.TryGetJoint(TrackedHandJoint.Palm, Handedness.ToXRNode().Value, out HandJointPose palmPose))
+                    HandSubsystem.TryGetJoint(TrackedHandJoint.Palm, Handedness.ToXRNode().Value, out HandJointPose palmPose) &&
+                    HandSubsystem.TryGetJoint(TrackedHandJoint.IndexProximal, Handedness.ToXRNode().Value, out HandJointPose knucklePose))
                 {
-                    // If prompted to use the ray vector, this is pose is calculated by simulating a hand ray initialized at the device pose.
-                    // This occurs when the simulation mode is set to ArticulatedHand
-                    handRay.Update(PlayspaceUtilities.ReferenceTransform.TransformPoint(simulatedControllerState.devicePosition), -palmPose.Up, Camera.main.transform, Handedness);
-                    Ray ray = handRay.Ray;
-                    simulatedControllerState.pointerPosition = PlayspaceUtilities.ReferenceTransform.InverseTransformPoint(ray.origin);
-                    simulatedControllerState.pointerRotation = Quaternion.Inverse(PlayspaceUtilities.ReferenceTransform.rotation) * Quaternion.LookRotation(ray.direction);
+                    // If prompted to use the ray vector, this pose is calculated by simulating a hand ray.
+                    // This occurs only when the simulation mode is set to ArticulatedHand.
+                    handRay.Update(
+                        knucklePose.Position,
```

Comment:
If I remember correctly, we weren't able to use the knuckle for simulation in the past due to the instability of the knuckle during gesture animations, where (at least for MRTK2), the index tip was the most-stable point. Are we adjusting (or have we adjusted) the simulation to stabilize the knuckle instead?